File: Linnworks/src/net/LinnworksAPI/OrderWorkflow.cs
using System.Text;
using System;
using System.Collections.Generic;
using Newtonsoft.Json.Converters;
using Newtonsoft.Json;
using System.Net.Http;
using System.Drawing;
using System.Collections.ObjectModel;
using System.Xml;
using System.ComponentModel;
using System.Reflection;

namespace LinnworksAPI
{
	public static class OrderWorkflowMethods 
	{ 
		private static JsonSerializerSettings serializerSettings = new JsonSerializerSettings() { DateFormatString = "yyyy-MM-ddTHH:mm:ss.ffZ"};
 		public static CheckinUserResponse CheckinUser(CheckinUserRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<CheckinUserResponse>(Factory.GetResponse("OrderWorkflow/CheckinUser", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static DeallocateOrderFromJobResponse DeallocateOrderFromJob(DeallocateOrderFromJobRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<DeallocateOrderFromJobResponse>(Factory.GetResponse("OrderWorkflow/DeallocateOrderFromJob", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetGroupResponse GetGroup(GetGroupRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetGroupResponse>(Factory.GetResponse("OrderWorkflow/GetGroup", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetGroupListResponse GetGroupList(GetGroupListRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetGroupListResponse>(Factory.GetResponse("OrderWorkflow/GetGroupList", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetJobResponse GetJob(GetJobRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetJobResponse>(Factory.GetResponse("OrderWorkflow/GetJob", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetJobAuditResponse GetJobAudit(GetJobAuditRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetJobAuditResponse>(Factory.GetResponse("OrderWorkflow/GetJobAudit", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetJobResponse GetJobByName(GetJobByNameRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetJobResponse>(Factory.GetResponse("OrderWorkflow/GetJobByName", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetJobErrorsResponse GetJobErrors(GetJobErrorsRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetJobErrorsResponse>(Factory.GetResponse("OrderWorkflow/GetJobErrors", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetPrintAttachmentResponse GetPrintAttachment(GetPrintAttachmentRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetPrintAttachmentResponse>(Factory.GetResponse("OrderWorkflow/GetPrintAttachment", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static GetWorkflowResponse GetWorkflow(GetWorkflowRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<GetWorkflowResponse>(Factory.GetResponse("OrderWorkflow/GetWorkflow", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static RunResponse Run(RunJobsRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<RunResponse>(Factory.GetResponse("OrderWorkflow/Run", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		}

		public static UpdateGroupResponse UpdateGroup(UpdateGroupRequest request,Guid ApiToken, String ApiServer)		{
			return Newtonsoft.Json.JsonConvert.DeserializeObject<UpdateGroupResponse>(Factory.GetResponse("OrderWorkflow/UpdateGroup", "request=" + Newtonsoft.Json.JsonConvert.SerializeObject(request, serializerSettings) + "", ApiToken, ApiServer), new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore}); 
		} 
	}
}

File: Linnworks/src/php/OrderWorkflow.php
<?php
class OrderWorkflowMethods 
{ public static function CheckinUser($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/CheckinUser", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function DeallocateOrderFromJob($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/DeallocateOrderFromJob", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetGroup($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetGroup", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetGroupList($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetGroupList", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetJob($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetJob", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetJobAudit($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetJobAudit", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetJobByName($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetJobByName", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetJobErrors($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetJobErrors", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetPrintAttachment($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetPrintAttachment", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function GetWorkflow($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/GetWorkflow", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function Run($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/Run", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
}

public static function UpdateGroup($request,$ApiToken, $ApiServer)
{
 return json_decode(Factory::GetResponse("OrderWorkflow/UpdateGroup", "request=" . json_encode($request) . "", $ApiToken, $ApiServer)); 
} 
}
?>